<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1040</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="TitleBar" name="titleBar" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="ToolBar" name="toolBar" native="true"/>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="templateBarHolder" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="VirtualRobotWidget" name="previewHolder" native="true"/>
        </item>
        <item>
         <widget class="JointAdjuestmentWidget" name="servoEditPane" native="true">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="behaviorEditHolder" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header>titlebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ToolBar</class>
   <extends>QWidget</extends>
   <header>toolbar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>JointAdjuestmentWidget</class>
   <extends>QWidget</extends>
   <header>jointadjuestmentwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VirtualRobotWidget</class>
   <extends>QWidget</extends>
   <header>virtualrobotwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
